tests/ui/test-attrs/tests-listing-format-json.rs

```diff
@@ -0,0 +1,18 @@
+// no-prefer-dynamic
+// compile-flags: --test
+// run-flags: --list --format json -Zunstable-options
```

Comment:
Please add a test showing that without -Zunstablre-opts we don't allow this.

Reply:
done